cmdstanpy/model.py

```diff
@@ -393,7 +387,7 @@ def optimize(
                 seed=seed,
                 inits=_inits,
                 output_dir=output_dir,
-                save_diagnostics=save_diagnostics,
+                save_diagnostics=False,
```

Comment:
This is not needed anymore?

Reply:
right - at the stan::services layer, https://github.com/stan-dev/stan/tree/develop/src/stan/services/optimize - if you look at these methods, they don't take a diagnostics file as an argument.